packages/app-store/zapier/components/zapierSetup.tsx

```diff
@@ -25,12 +26,15 @@ export default function ZapierSetup(props: IZapierSetupProps) {
   const integrations = trpc.useQuery(["viewer.integrations"]);
   const oldApiKey = trpc.useQuery(["viewer.apiKeys.findKeyOfType", { appId: ZAPIER }]);
   const deleteApiKey = trpc.useMutation("viewer.apiKeys.delete");
+  const appKeys = trpc.useQuery(["viewer.appKeys", { slug: "zapier" }]);
```

Comment:
We should never query `appKeys` in the FrontEnd. This is a serious security risk for leaking production API keys. If you only need the `invite_link` let's fetch and pass this only on getStaticProps. No need for tRPC in here.

Reply:
Thank you! I'll do that